Mathlib/AlgebraicTopology/SimplicialObject.lean

```diff
@@ -23,6 +23,7 @@ Use the notation `X _[n]` in the `Simplicial` locale to obtain the `n`-th term o
 
 set_option autoImplicit true
 
+/- -/
```

Comment:
Can you remove this comment? Or did you mean to write something here.

Reply:
Totally forgot to remove it, thank you :)